addons/l10n_latam_invoice_document/wizards/account_move_reversal.py

```diff
@@ -32,15 +39,15 @@ def _reverse_type_map(self, move_type):
             'in_receipt': 'out_receipt'}
         return match.get(move_type)
 
-    @api.depends('l10n_latam_available_document_type_ids')
+    @api.depends('l10n_latam_available_document_type_ids', 'journal_id')
     def _compute_document_type(self):
         for record in self.filtered(
                 lambda x: not x.l10n_latam_document_type_id or
                 x.l10n_latam_document_type_id not in x.l10n_latam_available_document_type_ids):
             document_types = record.l10n_latam_available_document_type_ids._origin
             record.l10n_latam_document_type_id = document_types[0] if document_types else False
 
-    @api.depends('move_ids', 'journal_id')
+    @api.depends('l10n_latam_available_document_type_ids', 'journal_id')
     def _compute_documents_info(self):
         self.l10n_latam_available_document_type_ids = False
         self.l10n_latam_use_documents = False
```

Comment:
_compute_documents_info is not working ok, but if you edit line 70 I think will be resolved
```diff
-               record.l10n_latam_use_documents = record.move_ids.journal_id.l10n_latam_use_documents
+               record.l10n_latam_use_documents = record.journal_id.l10n_latam_use_documents
```